bower.json

```diff
@@ -19,6 +19,7 @@
     "bower_components"
   ],
   "dependencies": {
-    "webcomponentsjs": "^0.7.22"
+    "webcomponentsjs": "^1.0.0",
+    "document-register-element": "^1.7.0"
```

Comment:
Why do we need `document-register-element`?

Reply:
Well.. that's because you are using Custom Element v0 syntax, I'd rather suggest to change it to CE v1, and stick to webcomponentsjs polyfill for v1 only.